src/InstrumentationEngine.NuGet/InstrumentationEngine.Nuget.csproj

```diff
@@ -63,6 +63,8 @@
       <Link>InstrumentationEngine\$(Platform)\Microsoft.Diagnostics.Instrumentation.Extensions.Base.pdb</Link>
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
+  </ItemGroup>
+  <ItemGroup Condition="'$(PlatformOS)'!='Linux'">
```

Comment:
Is arm64 only built for Windows at this time?

Reply:
That's correct